tools/topology/sof/pipe-smart-amplifier-playback.m4

```diff
@@ -69,6 +69,15 @@ C_CONTROLBYTES(Smart_amp Model, PIPELINE_ID,
         ,
         MODEL_priv)
 
+
+C_CONTROLBYTES_READONLY(Smart_amp Model_Get_params, PIPELINE_ID,
```

Comment:
@lgirdwood this change will make the tplg incompatible with older kernels. Should this be tied to ABI as well?

Reply:
yes. Added.